### PR TITLE
[Hotfix] Assign Volunteers UI Improvements

### PR DESCRIPTION
**Pull request by ngomichael**
### WIP

Still have to fix a small bug with copying link to clipboard button and add a hover effect to the cards, but otherwise everything else is pretty much done.

Just a heads up, there's some errors with Travis and how I formatted my code using Prettier, might need to update that or Prettier but either way works.

Trello: https://www.figma.com/file/0S0hMjYPnzhAvhnddxgEgGud/Food-Waste-DRG-Revision-3?node-id=1%3A3

### Changes:
Updated the Assign Volunteers page to bring it closer to the design spec and added some of my own designs.

### Next Step:
Start importing the css file and using CSS modules, SASS, and proptypes.

### CODE REVIEW

**Comment by jkbach** [comment]
Also, let's save the CSS module stuff for a separate PR, after Keller's stuff is approved and merged. We can definitely merge this and move on to other tasks in the meantime.

File: src/PageContent/AssignVolunteers/AssignVolunteersIndex.js [hunk 1]
```diff
@@ -3,47 +3,45 @@ import './AssignVolunteersIndex.css';
...
-                                    handleEditClick={handleEditClick} 
-                                    key={deliveryId} 
-                                    delivery={deliveries[deliveryId]} 
-                                    deliveryId={deliveryId}/>)
-                            
-                    } 
-                </div>
-            );
-        }
+    console.log(Object.keys(deliveries));
```
[the remark this answers is not in the document]
Comment (ngomichael) [reply src/PageContent/AssignVolunteers/AssignVolunteersIndex.js]: Sorry about the commented out styles and console logs, slipped my mind to do a quick scan before I made the PR

**Comment by ngomichael** [comment]
I think we can shorten the text in the button for now, but I can revisit the problem in the future and see what I can do to fix it.

**Comment by jkbach** [comment]
lgtm. Rebase and go ahead and merge.

**Pull request merged**